pkg/querybuilder/where_clause_visitor.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	"github.com/SigNoz/signoz/pkg/errors"
 	grammar "github.com/SigNoz/signoz/pkg/parser/grammar"
+	"github.com/SigNoz/signoz/pkg/query-service/constants"
```

Comment:
is it possible to define the constant somewhere in `https://github.com/SigNoz/signoz/tree/main/pkg/querybuilder`. We try to avoid importing from `pkg/query-service` in other `pkg/...`